### PR TITLE
[AAP-28597] Add Jira link to related skipped tests

### PR DESCRIPTION
**Pull request by akus062381**
This PR:
- deletes some unnecessary comments
- adds the related Jira link for AAP-28597 to the tests that were skipped

### CODE REVIEW

**Review by ShaiahWren — COMMENTED** [review]
The comment deletion looks good.
When running organizations.cy.ts  upstream, all but the first test fail consistently with a 401 error. Have you seen this?

File: cypress/e2e/awx/resources/inventories.cy.ts [hunk 1]
```diff
@@ -53,81 +52,67 @@ describe('Inventories Tests', () => {
...
-          cy.contains('button', 'Save inventory').click(); //Add an interception call for the edited inventory
+          cy.contains('button', 'Save inventory').click();
           cy.verifyPageTitle(inventory.name);
-          //Add assertions to verify the updated information is reflecting on the details screen of the edited inventory
-          cy.assertMonacoTextField('remote_install_path: /opt/my_app_config'); //Refactor this line to use the updated custom command
+          cy.assertMonacoTextField('remote_install_path: /opt/my_app_config');
           cy.hasDetail(/^Labels$/, label.name);
         });
 
         it('can copy an inventory on the details view and assert that the copy has been successful', () => {
```
Comment (ShaiahWren) [inline cypress/e2e/awx/resources/inventories.cy.ts:98]: intermittent failures on this one as well as `it('can delete an inventory from the inventory list toolbar', ()`
with error
```
get#filter
0
8
assertexpected #filter not to be disabled
```

**Comment by akus062381** [comment]
@ShaiahWren - I skipped the organizations spec file test that was failing. 
But the inventories test you mentioned- I could not get that one to fail?

**Review by ShaiahWren — APPROVED** [review]
Sometimes I have intermittent failures locally in orgs; let's see what the ci says.

**Review by ShaiahWren — APPROVED** [review]

**Pull request merged**